content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+
+    *type:* string
+    *default:*
+  * `region`
+    The region this subnet is in. See https://cloud.google.com/compute/docs/regions-zones/regions-zones
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    Optional SubNetwork name. The instance ID will be used by default.
+
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+
+
+
+### Example
+
+{{< gsHighlight  yaml  >}}
+
+my_net:
+  type: cloudify.gcp.nodes.Network
+  properties:
+    auto_subnets: false
+
+my_subnet:
+  type: cloudify.gcp.nodes.SubNetwork
+  properties:
+    subnet: 10.8.0.0/20
+  relationships:
+    - type: cloudify.gcp.relationships.contained_in_network
+      target: my_net
+
+my_instance:
+  type: cloudify.gcp.nodes.Instance
+  properties:
+    ...
+  relationships:
+    - type: cloudify.gcp.relationships.contained_in_network
+      target: my_subnet
+
+{{< /gsHighlight >}}
+
+If you want to use an exsisting SubNetwork (`use_external_resource: true`) then you must supply the `name` and `region` properties. This is because SubNetwork names are not unique across the whole project, only within a region.
+
+
+
+
+## cloudify.gcp.nodes.TargetProxy
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A TargetHttpProxy or TargetHttpsProxy.
+
+Specify which using the `target_proxy_type` property.
+
+
+
+**Properties:**
+
+
+  * `ssl_certificate`
+    URL of a SSL certificate associated with this target proxy. Can and must be used only with https type proxy.
+
+    *type:* string
+    *default:*
+  * `additional_settings`
+    Additional setting for target proxy
+
+    *default:* {}
+  * `name`
+    Optional target proxy name. By default it will be target proxy id.
+
+    *type:* string
+    *default:*
+  * `target_proxy_type`
+    This field indicates if this target proxy is a HTTP or HTTPS based target proxy. Possible values are: 'http' and 'https'.
+
+    *type:* string
+    *default:* http
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `url_map`
+    URL of a URL map which specifies how traffic from this target proxy should be redirected.
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.UrlMap
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+Maps URLs to BackendServices
+
+
+
+**Properties:**
+
+
+  * `default_service`
+    URL of a backend service to which this URL map will redirect traffic by default.
```

Comment:
**The** URL of a backend service to which this URL map will redirect traffic by default.